test/functional/pdo_sqlsrv/pdo_ae_insert_retrieve.phpt

```diff
@@ -0,0 +1,85 @@
+--TEST--
+Test for inserting encrypted data and retrieving both encrypted and decrypted data
+Retrieving SQL query contains encrypted filter
+--SKIPIF--
+<?php require('skipif_versions_old.inc'); ?>
+--FILE--
+<?php
+include 'MsSetup.inc';
+include 'MsCommon.inc';
+include 'AEData.inc';
+
+try
+{
+    $conn = ae_connect();
+    $conn->setAttribute( PDO::ATTR_ERRMODE, PDO::ERRMODE_SILENT );
+
```

Comment:
why errmode silent?

Reply:
Some tests expects an error and I want those tests to continue executing. This test however shouldn't expect an error. I'll change it.